vmdb/db/migrate/20150630100251_namespace_ems_amazon.rb

```diff
@@ -0,0 +1,75 @@
+class NamespaceEmsAmazon < ActiveRecord::Migration
```

Comment:
@matthewd I missed this in the other PR, but all data migrations should have a migration spec (see any of them in spec/migrations).  We probably should create one for the other PR as well when we get a chance.